enterprise-suite/gotests/testenv/testenv.go

```diff
@@ -141,7 +141,7 @@ func InitEnv() {
 	}
 
 	PrometheusAddr = fmt.Sprintf("%v/service/prometheus", ConsoleAddr)
-	MonitorAPIAddr = fmt.Sprintf("%v/service/es-monitor-api", ConsoleAddr)
```

Comment:
Might there be value in continuing to test the old endpoint as well for now, especially given these changes?

Reply:
Sure - I will add a basic backwards compatibility check.